x/oracle/spec/02_state.md

```diff
@@ -0,0 +1,61 @@
+<!--
+order: 2
+-->
+
+# State
+
+## ExchangeRate
+
+An `sdk.Dec` that stores an exchange rate against USD, which is used by the [Leverage](../../leverage/spec/README.md) module.
+
+- ExchangeRate: `0x01<denom_Bytes> -> amino(sdk.Dec)`
+
+## FeederDelegation
+
+An `sdk.AccAddress` (`umee-` account) address of `operator`'s delegated price feeder.
+
+- FeederDelegation: `0x02<valAddress_Bytes> -> amino(sdk.AccAddress)`
+
+## MissCounter
+
+An `int64` representing the number of `VotePeriods` that validator `operator` missed during the current `SlashWindow`.
+
+- MissCounter: `0x03<valAddress_Bytes> -> amino(int64)`
```

Comment:
```suggestion
- MissCounter: `0x03 | lengthPrefixed(valAddress) -> protobuf(uint64)`
```
A gogoproto struct seems to be used for this one (see keeper.go line ~238). Not sure if `protobuf(uint64)` is the right way to indicate that, but we can get rid of the `amino`